samples/matter/template/README.rst

```diff
@@ -113,6 +113,28 @@ Device Firmware Upgrade support
     :start-after: matter_door_lock_sample_build_with_dfu_start
     :end-before: matter_door_lock_sample_build_with_dfu_end
 
+.. matter_template_nrf54l15_build_with_dfu_start
+
+The Device Firmware Upgrade (DFU) for the nRF54L15 PDK is exclusively available for the ``release`` build configuration and is limited to the use of internal MRAM for storage.
+This means that both the currently running firmware and the new firmware to be updated must be stored within the device's internal flash memory.
+Currently, there is no support for utilizing external flash memory for this purpose.
+
+To build the sample with DFU support, use the ``-DFILE_SUFFIX=release_internal`` flag in your CMake build command.
+
+The following is an example command to build the sample with support for OTA DFU only:
+
+.. code-block:: console
+
+    west build -b nrf54l15pdk_nrf54l15_cpuapp -- -DFILE_SUFFIX=release_internal
```

Comment:
Do we want to use `FILE_SUFFIX`? I mean, other configurations were not ported yet, so maybe let's stick with the `CONF_FILE` solution to be consistent? Supporting two different ways of configuration for different boards will be slightly confusing.

Reply:
OK, we can use CONF_FILE, you're right, we should be consistent even if FILE_SUFIX works here though.